src/Tizen.NUI/src/public/Application/NUIApplication.cs

```diff
@@ -592,6 +592,29 @@ static public Size GetScreenSize()
             return ret;
         }
 
+        /// <summary>
+        /// Sets the geometry hit-testing enabled or disabled for the application.
+        /// </summary>
+        /// <param name="enable">True to enable geometry hit-testing, false otherwise.</param>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        static public void SetGeometryHittestEnabled(bool enable)
+        {
+            Interop.Application.SetGeometryHittestEnabled(enable);
+            if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
+        }
+
+        /// <summary>
+        /// Checks whether geometry hit-testing is enabled for the application.
+        /// </summary>
+        /// <returns>True if geometry hit-testing is enabled, false otherwise.</returns>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        static public bool IsGeometryHittestEnabled()
+        {
+            return Interop.Application.IsGeometryHittestEnabled();
```

Comment:
```suggestion
            bool ret = Interop.Application.IsGeometryHittestEnabled();
            NDalicPINVOKE.ThrowExceptionIfExist();
            return ret;
```

Below throw code become dead. I hope we'd better use the codes like this.